layouts/partials/header.html

```diff
@@ -25,24 +25,30 @@
                     {{ $active := or (eq $currentPage.Title .Name) (or ($currentPage.HasMenuCurrent "main" .) ($currentPage.IsMenuCurrent "main" .)) }}
                     {{ if .HasChildren }}
                         <li class="main-navigation-links-item">
-                            <a {{ if ne .URL "" }}href="{{ .URL }}"{{end}} class="main-navigation-links-link has-dropdown {{ if $active }}active{{ end }}">
+                            <a {{ if ne .URL "" }}href="{{ .URL | relLangURL  }}"{{end}} class="main-navigation-links-link has-dropdown {{ if $active }}active{{ end }}">
                                 <span>{{ .Name }}</span>
                                 {{ partial "icon.html" "dropdown-arrow" }}
                             </a>
 
                             <ul class="main-navigation-links-dropdown">
                                 {{ range .Children }}
                                     <li class="main-navigation-links-dropdown-item">
-                                        <a href="{{ .URL }}" class="main-navigation-links-link">{{ .Name }}</a>
+                                        <a href="{{ .URL | relLangURL }}" class="main-navigation-links-link">{{ .Name }}</a>
                                     </li>
                                 {{ end }}
                             </ul>
                         </li>
                     {{ else }}
                         <li class="main-navigation-links-item">
+                          {{ if eq .Name "Get involved" }}
```

Comment:
This seems like a very delicate check. Why can't we just add `| relLangURL` like you did above without a check?

Reply:
because the “get involved” is not translated, just add ` | relLangURL` will caused “404 not found” 
if add ` | relLangURL` && copy “get involved” to `content/zh` caused test failed，because the pages not translated
I have thought about using `config.toml - Menu Params` to solve the problem 
e.g. 
`{{ with .Params.translated}}`
`[[menu.main]]
.....
[menu.main.params]
  translated = true`
but It’s a new feature in hugo 0.79.0
so i used it did a temporary fix。
it will be removed after “get involved” translated

Reply:
update

Reply:
get-involved translation [https://github.com/istio/istio.io/pull/9520](url)